api/channels/bootstrap.py

```
@@ -212,6 +212,14 @@ async def _start_channel(running: dict, account_id: str, channel: str, credentia
         await ch.start()
     except Exception as ex:
         LOGGER.error("failed to start chat channel %s (%s): %s", account_id, channel, ex)
+        try:
+            await ch.stop()
+        except Exception as cleanup_ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)